lib/aws/session_store/dynamo_db/configuration.rb

```diff
@@ -12,7 +12,7 @@
 # language governing permissions and limitations under the License.
 
 require 'yaml'
-require 'aws-sdk-v1'
+require 'aws-sdk'
 
 module AWS::SessionStore::DynamoDB
```

Comment:
Thing I'm unsure about: If we should change the top-level module to be `Aws` to match the nomenclature across other such libraries. There are valid arguments to both sides of this question.